src/main/java/core/basesyntax/service/impl/ReturnFruitOperation.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dto.FruitDto;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.FruitOperations;
+import java.util.Map;
+
+public class ReturnFruitOperation implements FruitOperations {
+    private Map<String, Fruit> storage = Fruit.getFruitStorage();
+
+    @Override
+    public void doOperation(FruitDto fruitDto) {
+        String fruitName = fruitDto.getFruitName();
+        Integer amountToReturn = fruitDto.getAmount();
```

Comment:
these lines are duplicated in all classes that implement FruitOperations. Maybe it's better to pass a Fruit instead of FruitDto?